tools/advisor/advisor/ini_parser.py

```diff
@@ -62,7 +62,7 @@ def get_element(line):
     def get_key_value_pair(line):
         line = line.strip()
         key = line.split('=')[0].strip()
-        value = line.split('=')[1].strip()
+        value = "=".join(line.split('=')[1:])
         if not value:
```

Comment:
In what case value could be None?

Reply:
I added this check for checking for empty string, for example in the OPTIONS file we can have a line:
"db_log_dir="
Changing the if-condition to explicitly check for empty string for clarity